### ISSUE TITLE
Docs: Script tag incorrectly placed in learning guide

### ISSUE THREAD

**Issue by dswersky** [vercel/next.js#54303]
### What is the improvement or update you wish to see?

TL;DR: Script component markup should *not* be included in Head tag

This lesson introduces the Script component:
https://nextjs.org/learn/basics/assets-metadata-css/third-party-javascript

The lesson instructs learners to include the Script component within the Head tag. Tried this but it didn't work- the script was not loaded, the console.log didn't report and window.FB was undefined.

Found this StackOverflow: https://stackoverflow.com/questions/68058814/next-11-and-adding-script-tags-not-working-no-scripts-are-rendered

Followed the directions (moved the Script tag outside Head) and it works. 

### Is there any context that might help us understand?

No, that's it

### Does the docs page already exist? Please link to it.

https://nextjs.org/learn/basics/assets-metadata-css/third-party-javascript
